__tests__/components/Button.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Button } from '../../src/components';
+
+describe('Button', () => {
+  const onPress = { navigate: jest.fn() };
```

Comment:
you should declare a function instead of an object here for `onPress` to avoid the prop warnings. if we would pass an object inside of the app, the buttons would not work on press.
Button from react native requires a function https://reactnative.dev/docs/button#requiredonpress

Reply:
thanks! done in 9d3b4be